### ISSUE TITLE
Upgrade to 2.0.2 broke all components for me

### ISSUE THREAD

**Issue by OliverGrimsley** [monterail/vue-multiselect#422]
My app was on 2.0.0-beta.15, working fine.  I started updating all my scripts, and this upgraded to 2.0.2 - which broke all the multiselects - async, straight single selects, multi-selects and taggable ones - everything stopped working.  In the network tab in dev tools, my API was getting the data (checked async data from API, and objects API) - it looked like after the initial data was sent in, then the script emptied the data out for some reason.

I did not see any documentation updates as to breaking changes for my API - maybe I missed it.

I deleted my node-modules, rebuilt the entire repo, this did not fix it.  I reverted back to 2.0.0-beta.15 and now it works fine the way it did.  I can try and figure this out but under a crunch on some other items so I will have to remain at 2.0.0-beta.15 until I have time.  

Wondering if anyone else has had the same issue and could point me to what changed between 2.0.0-beta.15 and 2.0.2 that causes this to happen or what I need to change?

**Comment by michaelycus** [comment]
Same thing for me. I spent more than one day trying to find a way to make version 2.0.2 works but not success until now.
I'm also thinking to go back to version 2.0.0-beta.15.

**Comment by tonila** [comment]
I had to include vue-multiselect/dist/vue-multiselect.min.css and things started to work.

**Comment by OliverGrimsley** [comment]
I already was calling that CSS statically in my page load using the beta version, which was working fine and has been required I think for a bit of time now.  I have no custom CSS overrides either. The docs say you can call it statically or in the component ref.  I should have mentioned, 2.0.2 was not throwing any errors.  It seems to see the data from the API call, but then it deletes it automatically before rendering the element.   I will try again this weekend and see if I can figure it out.  I will also try dynamic loading CSS in the component, just in case.

**Comment by shentao** [comment]
Sorry for my lack of involvement here. Digging through the issues just yet. Hopefully, the problem was solved. If not, please reopen.

**Comment by OliverGrimsley** [comment]
I should have caught on what @tonila said.  I upgraded again and noticed the dropdowns were not showing again in _some_ components - and then I caught that I was using the `<style></style>` in my vue components where it was not working.  I had moved the CSS to static on my page for later components.  Removing all of the `<style></style>`  tags in my vue components brought everything back as then the CSS was just obtained from the static CSS.  All is good.  Thx for your work on the component. / mike
